src/components/box/box.component.tsx

```diff
@@ -24,10 +24,13 @@ export type BoxSizing = "content-box" | "border-box";
 
 type DesignTokensType = keyof typeof DesignTokens;
 type BoxShadowsType = Extract<DesignTokensType, `boxShadow${string}`>;
-export type BorderRadiusType = Extract<
-  DesignTokensType,
-  `borderRadius${string}`
->;
+type BorderRadiusToken = Extract<DesignTokensType, `borderRadius${string}`>;
+
+export type BorderRadiusType =
```

Comment:
suggestion: this type could be shared between this and PopoverContainer to ensure consistency